Resources/Prototypes/_Harmony/Magic/projectile_spells.yml

```
@@ -0,0 +1,24 @@
+﻿- type: entity
+  id: ActionWehball
+  name: Weball
+  description: Fires an weh towards the clicked location.
```

Minor spelling mistake
```suggestion
  description: Fires a weh towards the clicked location.
```